src/streaming/controllers/ScheduleController.js

```diff
@@ -253,12 +268,16 @@ function ScheduleController(config) {
 
     function onQualityChanged(e) {
         if (type !== e.mediaType || streamProcessor.getStreamInfo().id !== e.streamInfo.id) return;
-
         currentRepresentationInfo = streamProcessor.getRepresentationInfoForQuality(e.newQuality);
         if (currentRepresentationInfo === null || currentRepresentationInfo === undefined) {
             throw new Error('Unexpected error! - currentRepresentationInfo is null or undefined');
         }
 
+        if (e.oldQuality !== -1) { //blocks init quality change
```

Comment:
NIT: It would be cool if this magic number was a CONSTANT exported from AbrController and used here and in AbrController.